db/migrate/20180910225753_create_job_runs.rb

```diff
@@ -0,0 +1,10 @@
+class CreateJobRuns < ActiveRecord::Migration[5.2]
+  def change
+    create_table :job_runs do |t|
+      t.string :output_location
+      t.integer :job_type
```

Comment:
totally fine to see this done in a follow on ticket, but `job_type` should probably be required at both the schema level and the model level (i.e. a not null constraint here and a `presence: true` validation in the model).